### ISSUE TITLE
Include XVG

### ISSUE THREAD

**Issue by treygriffith** [treygriffith/swap-compatibility#6]
Source: https://twitter.com/CryptoRekt/status/958034085422575616

Things I need:
- Supported hash functions (Bitcoin derivatives typically include `RIPEMD160`, `SHA1`, and `SHA256`) and a link to the code that supports them
- Transaction malleability fix status (e.g. is SegWit on mainnet) and supporting evidence
- Support for Hashed Time Locked Contracts (typically by supporting `OP_CHECKLOCKTIMEVERIFY`, introduced in BIP65) and a link to supporting code
- Link to the software implementation that shows XVG in an atomic swap and its current status

**Comment by gummideepak** [comment]
@justinvforvendetta Could you give this a look!?

**Comment by justinvforvendetta** [comment]
yep tmrw I can

**Comment by jonathancross** [comment]
@justinvforvendetta ?

**Comment by justinvforvendetta** [comment]
@jonathancross yeah, since this issue was opened 2 years ago, we rebased with bitcoin 0.18 (now almost 0.19), so you can definitely go ahead with it now. we do not use segwit.
